src/gui/reports/ReportsWidgetHibp.cpp

```diff
@@ -157,16 +186,24 @@ void ReportsWidgetHibp::makeHibpTable()
             title.append(tr(" (Excluded)"));
         }
 
+        QStandardItem *titleItem, *pathItem, *exposedItem;
         auto row = QList<QStandardItem*>();
-        row << new QStandardItem(entry->iconPixmap(), title)
-            << new QStandardItem(group->iconPixmap(), group->hierarchy().join("/"))
-            << new QStandardItem(countToText(count));
+        titleItem = new QStandardItem(entry->iconPixmap(), title);
+        titleItem->setData(title, Qt::UserRole);
+        row << titleItem;
+        pathItem = new QStandardItem(group->iconPixmap(), group->hierarchy().join("/"));
+        pathItem->setData(group->hierarchy().join("/"), Qt::UserRole);
+        row << pathItem;
+        exposedItem = new QStandardItem(countToText(count));
+        exposedItem->setData(count, Qt::UserRole);
+        row << exposedItem;
```

Comment:
With ReportSortProxyModel we now sort rows based on their data and not on their Qt::UserRole.
Do we still need each column to have a setdata with Qt::UserRole?
```suggestion
        row << new QStandardItem(entry->iconPixmap(), title)
            << new QStandardItem(group->iconPixmap(), group->hierarchy().join("/"))
            << new QStandardItem(countToText(count));
```